src/components/UiText/index.vue

```diff
@@ -1,6 +1,10 @@
 <template>
-  <div class="ui-text" :for="inputId">
-    <span class="ui-text__dock ui-text__dock_left" v-if="hasSlot('dock-left')">
+  <div
```

Comment:
I'll update `ui-text` component soon, so please rollback these changes to avoid conflicts on merge